SpaceEnvironment_TotalLaunches/js/area.js

```diff
@@ -0,0 +1,82 @@
+$(function() {
+  $('#hcContainer').highcharts({
+    // Load Data in from Google Sheets
+    data: {
+      googleSpreadsheetKey: '1FpDY2RRA1EW5Pk7cK9tRSstaH6P0f-2GEDc_VjNRI4o',
+      googleSpreadsheetWorksheet: 1,
+      complete: function (data) {
+        // Display legend alphabetically
+        let dataItems = data.series
+        let dataNames = []
+
+          dataItems.forEach(item => {
+              item.legendIndex = ''
+              dataNames.push(item.name)
+            })
+
+            notSorting = dataNames.filter(arrayItem => arrayItem === 'Others')
+            alphaSorting = dataNames.filter(arrayItem => arrayItem !== 'Others')
+            alphaSorting.sort().push(notSorting[0])
+
+            dataItems.forEach(item => {
+              for (i = 0; i < alphaSorting.length; i++) {
```

Comment:
Rather than looping through `alphaSorting`, you could use the [.findIndex()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex) method based on the name.

Another potential change would be to convert the `.forEach` to a `.map`, but that isn't necessary (and the `forEach` is likely faster, performance-wise).